src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -121,4 +127,18 @@ private SearchFilterFiles() {}
 	private static String getFileName( final Path path ) {
 		return path.getFileName().toString();
 	}
+
+	public static Filter getFilterByName( final String name ) {
```

Comment:
Oh, just realized. Do you want to create two quick and simple tests, that are not integration tests?

Reply:
To test what? I adapted one of the existing tests, to make sure .js files are included: https://github.com/retest/recheck/pull/394/commits/22d1673d57b70306ebbbb7c042a306c9df5acc7f

Reply:
Like explicitly test this method that it finds a filter or throws an exception. The only tests are when `RecheckOptions` are created.

Reply:
Done